### DISCUSSION TITLE
Reverification of Student pack rejected

### DISCUSSION THREAD

**Discussion by harshitk024** [community/community#52250] [GitHub Education]
I applied for student developer pack , after 5 unsuccessful attempts  i got approved . But after 2 days it asked me to re-verify my application as i went to re-verify my student pack , in school section i got this message -  

<img width="435" alt="error " src="https://user-images.githubusercontent.com/107901497/230600870-0573a65e-0db2-48f3-91ea-2f67f33703d6.png">

As far as i know , it is saying me to use this email address but this is not my school associated email address . My primary email is the only one which is associated with my college but it does not let me use that ,  I don't know what to do , i tried almost 5 times but i don't know what's the solution if anyone know about it please help me. 
Thanks

**Comment by samus-aran** [comment]
Hi @Harshitk024 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

You are also stating that the school you've selected isn't the school domain or are you attending another school?
